system-x/services/azure/key-vault/src/main/java/software/tnb/azure/keyvault/client/AzureKeyVaultClient.java

```diff
@@ -0,0 +1,35 @@
+package software.tnb.azure.keyvault.client;
+
+import software.tnb.azure.keyvault.account.KeyVaultAccount;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.azure.identity.ClientSecretCredential;
+import com.azure.identity.ClientSecretCredentialBuilder;
+import com.azure.security.keyvault.secrets.SecretClient;
+import com.azure.security.keyvault.secrets.SecretClientBuilder;
+
+public class AzureKeyVaultClient {
```

Comment:
this class isn't needed, since it only creates a client - you should override `client()` method in `KeyVault` class to create and return the client instance (for example https://github.com/tnb-software/TNB/blob/main/system-x/services/azure/service-bus/src/main/java/software/tnb/azure/service/bus/service/ServiceBus.java#L15-L21 )

Reply:
Removed this class.